src/v0/destinations/tiktok_ads_offline_events/config.js

```diff
@@ -23,8 +23,17 @@ const EVENT_NAME_MAPPING = {
   contact: 'Contact',
   submitform: 'SubmitForm',
   subscribe: 'Subscribe',
+  'product added to wishlist': 'AddToWishlist',
```

Comment:
better to follow alphabetical order for keys

Reply:
Done